community-plugins.json

```diff
@@ -9,12 +9,23 @@
     "memories": true,
```

Comment:
Pls remove changes to this file, your formatting is different.

Reply:
Oops, didn't notice - will fix

Reply:
fixed